python/doc/developer_guide/architecture.rst

```diff
@@ -190,7 +190,7 @@ The tools chosen for the development of the platform are:
 +---------------------------------------+---------------------------------------------------------------------------------+-------------------+
 | Special functions (optional)          | `MPC <https://www.multiprecision.org/mpc>`_                                     | 1.1.0             |
 +---------------------------------------+---------------------------------------------------------------------------------+-------------------+
-| Optimization (optional)               | `NLopt <https://nlopt.readthedocs.io/en/latest/>`_                              | 2.6               |
+| Optimization                          | `NLopt <https://nlopt.readthedocs.io/en/latest/>`_                              | 2.6               |
```

Comment:
I have scenarios where nlopt is too old than 2.6, like the Mageia 9 distribution , so that won't work for me